examples/kubernetes/1.10/cilium-crio-ds.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         - name: clean-cilium-state
           image: docker.io/library/busybox:1.28.4
           imagePullPolicy: IfNotPresent
-          command: ['sh', '-c', 'if [ "${CLEAN_CILIUM_STATE}" = "true" ]; then rm -rf /var/run/cilium/state; rm -rf /sys/fs/bpf/tc/globals/cilium_*; fi']
+          command: ['sh', '-c', 'rm /var/run/cilium/{cilium.pid,state/health-endpoint.pid}; if [ "${CLEAN_CILIUM_STATE}" = "true" ]; then rm -rf /var/run/cilium/state; fi; if [ "${CLEAN_CILIUM_STATE}" = "true" ] || [ "${CLEAN_CILIUM_BPF_STATE}" = "true" ]; then rm -rf /sys/fs/bpf/tc/globals/cilium_* /var/run/cilium/bpffs/tc/globals/cilium_*; fi']
```

Comment:
Now it's the time to create a script.. please? Or even container based in busybox to run those commands. `docker.io/cilium/clean-state`

Can you create a GH issue to track this?